cumulus/constants.py

```diff
@@ -18,14 +18,20 @@
 ###############################################################################
 
 
+from enum import IntEnum
+
+
 class ClusterType:
     EC2 = 'ec2'
+    ANSIBLE = 'ansible'
     TRADITIONAL = 'trad'
```

Comment:
I do have concerns about introducing a cluster type of 'ansible'.  In the future when we support 'openstack' and 'ec2' how will be distinguish between the two? May be we need the concept of a cluster "provisioner"